posts/2020/08/but-what-are-traitlets.md

```diff
@@ -0,0 +1,305 @@
+<!--
+.. title: But What are Traitelts ?
+.. slug: what-are-traitlets
+.. date: 2020-08-xx 22:00:00 UTC-00:00
+.. author: Matthias Bussonnier
+.. tags: Labs, IPython, traitlets, historical
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+You probably have head the name, you likely have used them, but you likely are
+still confused ? What are `trailtlets`; in the following we'll discuss a bit
```

Comment:
can you check the spelling of `traitlets` in the whole post? it's wrong here and in the title at least, maybe elsewhere